src/server.js

```diff
@@ -113,21 +77,20 @@ var RdHandler = {
   },
 
   /**
-   * Handler for incoming requests to Requests Debugger Tool proxy server.
+   * Handler for incoming requests to Requests Debugger Tool server.
    * @param {http.IncomingMessage} clientRequest 
    * @param {http.ServerResponse} clientResponse 
    */
   requestHandler: function (clientRequest, clientResponse) {
     clientRequest.id = ++RdHandler._requestCounter + '::' + uuidv4();
-
+    var uri = url.parse(clientRequest.url).path;
     var request = {
       method: clientRequest.method,
       url: clientRequest.url,
       headers: clientRequest.headers,
       data: []
     };
-    
-    RdGlobalConfig.reqLogger.info(constants.TOPICS.CLIENT_REQUEST_START, request.method + ' ' + request.url,
+    RdGlobalConfig.reqLogger.info(constants.TOPICS.CLIENT_REQUEST_START, request.method + ' ' + uri,
```

Comment:
In case of using it in proxy mode... we would be logging just the `path` and not the whole URL?

Reply:
Similarly at all other places..

Reply:
Its for consistency in logs which will help user to debug.

Reply:
Changed to whole url

Reply:
can be resolved.